index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+  </head>
+
+  <body>
+
+    <nav>
+      <ul>
+        <li><a href="#welcome-section"> Welcome </a></li>
+        <li><a href="#works">Work</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
```

Comment:
- [ ] I think it would be better if you wrap the `<nav>` tag in `<header>` tag.

Reply:
>  I think it would be better if you wrap the `nav` tag in `header` tag.
> 
> * [ ]  Please always try to explain the reasoning for your suggestion. it may not be obvious to the reader why you are making this suggestion.
> 
> > Your implementation using nav tag.
> 
> * [ ]  Also try to be more clear in your suggestion about why it is better to wrap nav in the header using examples.